index.html

```diff
@@ -24,22 +25,6 @@
                title="Bazel"
                alt="Bazel logo"
                class="logo img-responsive" />
-          <p class="tagline">{Fast, Correct} - Choose two</p>
```

Comment:
I appreciate the detailed explanation, Steren. My main motivation for sending this PR is that I still feel that the placement of the tagline still appears awkward on the landing page, and I would like to propose some ideas to improve it.

Here is my thought process:

While we agreed on the contents of the landing page on the doc, the design of the landing page has, in fact, evolved considerably since then, especially after our UX consultations with the designers late last week. As mentioned during our recent meetings, the hero section of the landing page appears unbalanced, especially with the tagline underneath the logo, which appears to point to the tagline.

Because I saw that the landing pages of similar projects such as Gradle and those of well-known Google products such as Google Cloud and Adsense followed the pattern of making the tagline the most prominent text on the landing page, I implemented this change.

Some other thoughts:

> 1. ... These brands use their tagline at the end of their TV ads below their logos, not at the beginning.

I don't think this is an apples-to-apples comparison since Bazel is a single open source project and these are large, multi-national corporations with diverse portfolios of products. A more comparable example is the landing page for one of their products, such as the [Volkswagen Atlas](http://www.vw.com/models/atlas/), wherein the most prominent text after the name of the product is the tagline, "Life's as big as you make it."

Given that we want to give Bazel its standalone branding identity rather than making it a child of another project (similar to how each Cloud product is a child of the Google Cloud platform), the way I see it is that the Bazel landing page is more comparable to the landing page for Google Cloud itself, which is why I listed the Google Cloud website as an example, which displays the taglines "Build What's Next" and "Better software. Faster." most prominently.

> 2. My proposal was not to get rid of the Bazel tagline. I suggest to keep the tagline, but keeping is closer to the logo, closer to our brand.

I understand that. However, I feel that the way that the tagline is placed appears awkward and unbalanced, which is the main motivation for this PR.

> 3. I am fully open to iterate on the sentence. A GitHub PR is not the best medium to do so. Let's use Google Docs.

Agreed. If you'd like, I can create a wireframe version of this design on a doc for us to discuss.

> 4. Our designers recently suggested us to use https://www.firebase.com as a reference, as it is for them a perfect example of landing page. As you can see, they have this massive "Firebase helps you build better mobile apps and grow your business." at the top. They do not have any tagline.

That is true, and the wireframe that our designer sketched up did not have the tagline either. One option I listed above is to omit the tagline entirely, which would solve the problem of visual design, but on the other hand, many of us would like to keep the tagline, but the main question is what would be the best way to keep both?

After spending more time browsing http://beautifulopen.com/, though, I did manage to find an example of a landing page for an open source project that has both a tagline and a summary but with the summary more prominent than the tagline:

<a href="http://lattice.cf/" target="_blank"><img width="1470" alt="screen shot 2017-06-28 at 2 16 19 am" src="https://user-images.githubusercontent.com/5283042/27629689-046b42fc-5ba8-11e7-8142-8f138197b1d5.png"></a>

Though the project itself is being deprecated, I think that we can use a similar approach and basically invert the text sizes of the tagline and summary. Curiously, the graphic on Lattice's page is somewhat similar to our new logo, but I digress. :)

<img width="1470" alt="screen shot 2017-06-28 at 2 35 25 am" src="https://user-images.githubusercontent.com/5283042/27630628-e9caa2d2-5baa-11e7-8481-a39e303604df.png">

Here is a new revision of this design with the tagline stacked above the summary, but the summary is now the most prominent text in the hero. I think this might be a good compromise among 1) keeping the Bazel tagline 2) displaying the tagline together with the summary in a way that is visually pleasing 3) keeping the summary as the most prominent title when the user first views the page.

What do you think?